test/test_admin.py

```diff
@@ -17,6 +17,12 @@
 def admin(conn_string):
     with stardog.admin.Admin(**conn_string) as admin:
 
+        # IMO we should remove these. reason being, if by mistake a user decides to run this
```

Comment:
I don't have any strong opinions here.

Reply:
@buzztroll @rmyers thoughts? I would say this is dangerous and should be removed (probably on a different PR), and that we should only clean up resources created by this test suite.

Reply:
Can we name the database with something like `-testing-only` and just delete the databases by names? Then we shouldn't really have to worry about deleting prod data, unless someone magically chooses to name their DB's the exact same way.

I would also hope that the admin connection would fail on a prod db as well, otherwise that means they are running `admin:admin` in prod?